src/app/data/pontusx-account-names.ts

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { Layer } from '../../oasis-nexus/api'
+import { Network } from '../../types/network'
+import { hasTextMatch } from '../components/HighlightedText/text-matching'
+import * as process from 'process'
+import { AccountData, AccountEntry, AccountMap, AccountMetadataInfo } from './named-accounts'
+
+const DATA_SOURCE_URL = 'https://raw.githubusercontent.com/deltaDAO/mvg-portal/main/pontusxAddresses.json'
+
+const getPontusXAccountsMetadata = () =>
+  new Promise<AccountData>((resolve, reject) => {
+    axios.get(DATA_SOURCE_URL).then(response => {
+      if (response.status !== 200) reject("Couldn't load names")
+      if (!response.data) reject("Couldn't load names")
+      const map: AccountMap = new Map()
+      const list: AccountEntry[] = []
+      Object.entries(response.data).forEach(([address, name]) => {
+        map.set(address, { name: name as string })
+        const normalizedEntry: AccountEntry = {
+          name: name as string,
+          address,
+        }
+        list.push(normalizedEntry)
+      })
+      resolve({
+        map,
+        list,
+      })
+    }, reject)
+  })
+
+export const usePontusXAccountsMetadata = (enabled: boolean) => {
+  return useQuery(['pontusXNames'], getPontusXAccountsMetadata, {
+    enabled,
+    staleTime: Infinity,
+  })
+}
+
+export const usePontusXAccountMetadata = (address: string, enabled: boolean): AccountMetadataInfo => {
+  // When running jest tests, we don't want to load from Pontus-X.
+  if (process.env.NODE_ENV === 'test') {
+    return {
+      metadata: undefined,
+      loading: false,
+    }
+  }
```

Comment:
Same here. Mock globally in tests.

Reply:
OK, removed mocking from production code.